### DISCUSSION TITLE
Gibt es einen Beispiel-Datensatz zum Testen?

### DISCUSSION THREAD

**Discussion by fkuersch** [openjverein/jverein#455] [Q&A]
Moin moin, ich teste immer mal wieder neue Features etc. aus dem jVerein-Repo, bevor eine neue Version released wird. Dafür muss ich dann immer wieder von Hand neue Test-Daten eingeben (Mitglieder anlegen, Einstellungen vornehmen, Formulare erstellen etc.). Wie macht ihr das? Habt ihr Test-Daten, die ihr in die Datenbank importieren könnt o. Ä.?

**Comment by tolot27** [comment]
Moin. Ich spiele mir immer wieder eine Testdatenbank ein. Läuft bei mir auf MySQL und da lässt sich das einfach realisieren.

**Comment by dippeal** [comment]
Duplikat von https://github.com/openjverein/jverein/discussions/311
